src/states/elder/mod.rs

```diff
@@ -1137,6 +1154,35 @@ impl Base for Elder {
         &mut self.timer
     }
 
+    fn finish_handle_transition(&mut self, outbox: &mut dyn EventBox) -> Transition {
+        debug!("{} - State changed finish to Elder.", self);
```

Comment:
Weird wording. What about "State change to Elder finished" or just "State changed to Elder"?